src/main/kotlin/dk/cachet/carp/webservices/protocol/dto/ProtocolOverview.kt

```diff
@@ -0,0 +1,29 @@
+package dk.cachet.carp.webservices.protocol.dto
+
+import dk.cachet.carp.protocols.application.StudyProtocolSnapshot
+import java.time.Instant
+
+data class LatestProtocolOverview
+(
+    val ownerName: String?,
+    val snapshot: StudyProtocolSnapshot,
+    val lastVersionCreatedDate: Instant
+) {
+    /**
+     * The [GetLatestProtocolResponseDto.toLatestProtocolOverview()] function converts
+     * a data type [GetLatestProtocolResponseDto] to a [LatestProtocolOverview].
+     *
+     * @return A [LatestProtocolOverview] object containing the protocol from [snapshot].
+     */
+    companion object {
```

Comment:
dont do this this as an extension method, but instead do somethgin like `from(GetLatestProtocolResponseDTO): LatestProtocolOverview`.

extension methods of `GetLatestProtocolResponse` have little to do with this overview.

Reply:
okey